tests/main/interfaces-snap-interfaces-requests-control/api-client/bin/start

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+#!/bin/sh
+
+while true; do
+    echo "running"
+    sleep 10 || break
```

Comment:
Why `sleep 10 || break`, afaik sleep should always return 0, so this will never break?

Reply:
when systemd sends SIGTERM it will interrupt sleep and I want the whole script to exit at this point rather than possibly going into another iteration

In hindsight I should have used `set -e`, but maybe another time :smile: